pkg/controller/clusterstorage/clusterstorage_controller.go

```diff
@@ -89,6 +89,22 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to second resource CustomResourceDefinition and requeue the Infrastructure
+	// This is necessary to check if VolumeSnapshot CRDs are installed after the Operator
+	err = c.Watch(&source.Kind{Type: &apiextv1beta1.CustomResourceDefinition{}}, &handler.EnqueueRequestsFromMapFunc{
```

Comment:
nit: Why apiextv1beta1 and not apiextv1? We will need to update to v1 at some point.

Reply:
I've had numerous errors when attempting to put this as v1. I can try to sort through these, but I think we should do the upgrade in a separate commit.